File: Ashish/Week 1/plusone.java
public class Solution {
    public int[] plusOne(int[] digits) 
    {
        int length = digits.length;
        for(int i = length -1; i >= 0; i--)
        {
            if(digits[i] < 9)
            {
                digits[i] += 1;
                return digits;
            }
            digits[i] = 0;
        }
        int a[] = new int[length + 1];
        a[0] = 1;
        return a;
    }
}

File: Pranchal/week1/plusOne.py
class Solution:
    def plusOne(self, digits: List[int]) -> List[int]:
        length = len(digits)-1
        while length>-1:
            if digits[length]<9:
                digits[length]=digits[length]+1
                return digits
            else:
                digits[length]=0
                length = length-1
                print(length>-1)
        digits.insert(0,'1')
        return digits


#https://leetcode.com/problems/plus-one/